pt-BR/association_basics.md

```diff
@@ -537,16 +534,15 @@ class CreateEmployees < ActiveRecord::Migration[5.0]
 end
 ```
 
-Tips, Tricks, and Warnings
---------------------------
+## Tips, Tricks, and Warnings
```

Comment:
Acho que daqui pra baixo você pode pedir pro git ignorar essas mudanças pra não commitar o arquivo todo, o que acha?

Reply:
sim, provavelmente foi meu editor que fez essas mudanças :hankey: foi mal... existe algum comando git que posso usar?

Reply:
Acho que você pode voltar o seu commit pra deixar as alterações em "not staged" e dar um git add -p pra ir incluindo algumas linhas.

É claro que algum editor de git que não seja linha de comando vai te ajudar a fazer isso mais facilmente.

Reply:
valeu professor :smile: